### DISCUSSION TITLE
Syntax for sending a "MCU note" value - without notebutton or similar

### DISCUSSION THREAD

**Discussion by mwpmorris** [tttapa/Control-Surface#671] [Q&A] [answered]
Hi,

First of all, thanks for this amazing library. I've very nearly managed to finish my project - a small controller consisting of 2 x rotary encoders and a toggle switch talking to a MCU host.

The toggle switch cycles between 3 "modes" that the encoders are set to, 2 of which are utilising MCU VPOTs.

MODE 1 (MCU VPOT)-----MODE 2 (MCU PITCH BEND)------MODE 3 (MCU VPOT)

The issue I'm having is I can't figure out the syntax for sending a MIDI note with MCU:ASSIGN_TRACK for "MODE 1" above, and similarly for MCU:ASSIGN_PAN for "MODE 3".

I know this works as I can rig up a seperate test button and use:

`NoteButton encbutton1 {
	7,
	{MCU::ASSIGN_PAN},
};`

I need a way of using the MCU:ASSIGN_PAN note message without having it assigned to notebutton, so that I can use it in my regular buttonpress code - as I have a lot of other stuff happening there.

Hope that makes sense - many thanks!

**Comment by tttapa** [comment, marked as answer]
If you just want to send a MIDI note event, you can use `Control_Surface.sendNoteOn(address, velocity)`, see [MIDI_Sender::sendNoteOn()](https://tttapa.github.io/Control-Surface-doc/Doxygen/d6/d72/classMIDI__Sender.html#ad487d041b8e9d10d58aba470a15e1fc6).

For more information, see [MIDI tutorial: sending MIDI messages](https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/df7/midi-tutorial.html#midi_md-sending).

**Reply by mwpmorris** [reply]
That's it - thanks!
